src/style.css

```diff
@@ -1,3 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  background-color: #a09595;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  background-color: #fff;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.logo_image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav_list {
+  position: relative;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
+  line-height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.nav_item {
+  font-weight: 500;
+}
+
+li {
+  list-style: none;
```

Comment:
According to the checklist, you should avoid styling elements using tag names (except `html` and `body`). Instead, you should use class selectors to style your elements. In this case, you could give your `li` elements a class and style that class.